packages/backend/src/build-disk-image.ts

```diff
@@ -100,12 +100,13 @@ export async function buildDiskImage(build: BootcBuildInfo, history: History): P
   // "Returning" withProgress allows PD to handle the task in the background with building.
   return extensionApi.window
     .withProgress(
-      { location: extensionApi.ProgressLocation.TASK_WIDGET, title: 'Building disk image ' + build.name },
+      { location: extensionApi.ProgressLocation.TASK_WIDGET, title: 'Building disk image ' + build.id },
       async progress => {
-        const buildContainerName = build.name.split('/').pop() + bootcImageBuilderContainerName;
+        const buildContainerName = build.id + bootcImageBuilderContainerName;
```

Comment:
Ok to merge, but ditto here or maybe we find a non-conflicting name later. If there's no current container with a conflict I'd expect the name to be "image-name".